bitnami/postgresql/README.md

```diff
@@ -210,7 +210,7 @@ $ kubectl delete pvc -l release=my-release
 | `primary.priorityClassName`                   | Priority Class to use for each pod (postgresql primary)                                                                                                   | `""`                        |
 | `primary.extraInitContainers`                 | Extra init containers to add to the pods (postgresql primary)                                                                                             | `[]`                        |
 | `primary.extraVolumeMounts`                   | Extra volume mounts to add to the pods (postgresql primary)                                                                                               | `[]`                        |
-| `primary.extraVolumes`                        | Extra volumes to add to the pods (postgresql primary)                                                                                                     | `[]`                        |
+| `primary.extraVolumes`                        | Extra volumes to add to the pods (postgresql primary).                                                                                                    | `[]`                        |
```

Comment:
this point is not necessary. Please delete it. Please, leave the line as the original one (no changes, please).